testdriver/datasets.py

```diff
@@ -208,7 +208,8 @@ class ParallelMode(Enum):
   ParallelByLang = 2
 
 class NodeVersion(Enum):
-  Node20 = "20.1.0"
+  Node22 = "22.1.0"
+  Node20 = "21.6.0"
```

Comment:
Why does the `Node20` string begins with "21"? I don't see any usages of `Node22` in this PR, for example, so is that addition necessary, or can we remove `Node22`. And for that matter, where else does `NodeVersion` get used, or can we remove that altogether?

Reply:
Removing NodeVersion. Good catch!